### DISCUSSION TITLE
Drug Drug Interaction

### DISCUSSION THREAD

**Discussion by hkhadim** [Open-Systems-Pharmacology/Forum#1467] [Questions & Problems]
Dear researcher, I want to check drug drug interaction between anticancer regimen (having 4-5 drugs) with antiepileptic drug. kindly guide me about this.

**Comment by tobiasK2001** [comment]
Hi,

that sound like and interesting and probably complex project. You may find some inspiration in this publications dealing with multi compound DDIs done with the OSPs: 
https://www.mdpi.com/1999-4923/12/12/1191
https://doi.org/10.1007/s40262-019-00777-x
https://pubmed.ncbi.nlm.nih.gov/30091221/

Hope this might help you.

Best, Tobias

**Reply by hkhadim** [reply]
Dear Tobias ,Thank you so much for guide me...

**Comment by StephanSchaller** [comment]
Hi @hkhadim ,

you can check out tutorial/training on DDI with the OSP Suite on our online training platform: 

For Academia (free): https://learn-esqlabs.talentlms.com/
For Industry: https://esqlabs.shop/products/drug-drug-interactions-ddi-11

**Reply by hkhadim** [reply]
Dear StephanSchaller,
Thank you so much for this kindness...

**Comment by hkhadim** [comment]
Dear Researchers, can you guide me about clinical data of  these drugs in combination (Azathioprine with Allopurinol, Mercaptopurine with Febuxostat etc)
